blockchain/shardCoordinator_test.go

```diff
@@ -0,0 +1,34 @@
+package blockchain
+
+import (
+	"crypto/rand"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/data"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAddress_GetShard(t *testing.T) {
```

Comment:
can you rename to TestShardCoordinator_ComputeShardID ?

Reply:
renamed